src/solver/optimisation/opt_gestion_des_couts_couts_demarrage.cpp

```diff
@@ -25,13 +25,14 @@
 ** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
 */
 
-#include "opt_structure_probleme_a_resoudre.h"
+#include <antares/study/area/scratchpad.h>
```

Comment:
include the scratchpad ? Why ? Seems useless

Reply:
Was necessary but not after rebasing on develop (scratchpad deletion)